.github/workflows/mvn-verify-check.yml

```diff
@@ -187,27 +168,51 @@ jobs:
           }
           done
 
+  set-scala213-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      scala213Versions: ${{ steps.all213ShimVersionsStep.outputs.scala213Versions }}
+      sparkJDK17Versions: ${{ steps.all213ShimVersionsStep.outputs.jdkVersions }}
+    steps:
+      - uses: actions/checkout@v4 # refs/pull/:prNumber/merge
+
+      - id: all213ShimVersionsStep
+        run: |
+          set -x
+          SCALA_BINARY_VER=2.13
+          . jenkins/version-def.sh
+          svArrBodyNoSnapshot=$(printf ",{\"spark-version\":\"%s\",\"isSnapshot\":false}" "${SPARK_SHIM_VERSIONS_NOSNAPSHOTS[@]}")
+          svArrBodyNoSnapshot=${svArrBodyNoSnapshot:1}
+          # get private artifact version
+          privateVer=$(mvn help:evaluate -q -pl dist -Dexpression=spark-rapids-private.version -DforceStdout)
+          svJsonStr=$(printf {\"include\":[%s]} $svArrBodyNoSnapshot)
+
+          echo "scala213Versions=$svJsonStr" >> $GITHUB_OUTPUT
+
+          # jdk17
+          jdk17VersionArrBody=$(printf ",{\"spark-version\":\"%s\",\"java-version\":17}" "${SPARK_SHIM_VERSIONS_JDK17_SCALA213[@]}")
+
+          jdkVersionArrBody=$jdk17VersionArrBody
+          jdkVersionArrBody=${jdkVersionArrBody:1}
+          jdkVersionJsonStr=$(printf {\"include\":[%s]} $jdkVersionArrBody)
+          echo "jdkVersions=$jdkVersionJsonStr" >> $GITHUB_OUTPUT
+
   package-tests-scala213:
-    needs: cache-dependencies
+    needs: set-scala213-versions
```

Comment:
We need an issue how to better handle cache for scala2.13 build dependencies

Reply:
https://github.com/NVIDIA/spark-rapids/issues/11199